Data/Infrastructure/OJS.Data.Infrastructure/DateTimeValueConverter.cs

```diff
@@ -0,0 +1,43 @@
+namespace OJS.Data.Infrastructure;
+
+using Common.Utils;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using System;
+using System.Linq.Expressions;
+using static Common.GlobalConstants.EnvironmentVariables;
+
+public class DateTimeValueConverter : ValueConverter<DateTime?, DateTime?>
+{
+   private static readonly TimeZoneInfo LocalTimeZoneInfo = TimeZoneInfo.FindSystemTimeZoneById(EnvironmentUtils.GetRequiredByKey(LocalTimeZone));
```

Comment:
Add validation for the `LOCAL_TIME_ZONE` env variable in startup, like we have validation for ApplicationUrl for example in Ui. There is a helper method - `EnvironmentUtils.ValidateEnvironmentVariableExists`